cqf-fhir-cr/src/main/java/org/opencds/cqf/fhir/cr/measure/r4/R4CareGapsProcessor.java

```diff
@@ -166,6 +168,24 @@ private void checkMeasureBasis(Measure measure) {
         }
     }
 
+    /**
+     * MultiRate Measures require a unique 'id' per GroupComponent to uniquely identify results in Measure Report.
+     * This is helpful when creating DetectedIssues per GroupComponent so endUsers can attribute evidence of a Care-Gap to the specific MeasureReport result
+     * @param measure Measure resource
+     */
+    private void checkMeasureGroupComponents(Measure measure) {
+        // if a Multi-rate Measure, enforce groupId to be populated
+        if (measure.getGroup().size() > 1) {
```

Comment:
Suggestion:  Write a tiny utils class to wrap around group and use it everywhere you call getGroup().size() > 1

ex:

```java
MeasureGroupUtils.isMultiGroup(Measure measure) {...}
```

Reply:
agreed, we can upstream or separate this more efficiently using existing classes

MeasureDef=> isMultiGroup, 
GroupDef=> scoringType, improvementNotation

These classes are meant to hold the profile of the Measure that was evaluated. 

I'd like to address in a separate refactor since these values are used in several places, not just care-gaps